### PR TITLE
Updated the Swedish translation to sound more natural

### PR DESCRIPTION
**Pull request by cascading-jox**
The latest translation of the Swedish text that was merged was better than the last one but it still did not sound like something you would say in Swedish. I updated the translation once more with more commonly used English words instead of the literal translated words, restructured some sentences and made it overall sound more natural. I would appreciate some feedback from some Swedish talking people. 👍/👎?

### CODE REVIEW

File: i18n/sv.toml [hunk 1]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
+  other = "Rocky Linux är ett community-drivet enterprise-operativsystem designat för att vara 100% bugg-för-bugg kompatibelt med Amerikas främsta enterprise-klassade linuxdistribution nu när downstream-partnern har bytt inriktning. Rocky Linux är under intensiv utveckling av communityn. Det leds av Gregory Kurtzer, grundare av CentOS-projektet. Det är ännu inte bestämt när första versionen ska släppas. Ni som vill bidra uppmanas att kontakta oss med hjälp av de kommunikationsalternativ som erbjuds på webbplatsen."
+ 
+[faq]
   [faq.title]
   other = "Vanliga frågor"
 
   [faq.q]
     [faq.q.downstream-partner-direction]
-    other = "Vad menar ni med att \"dess nedströmspartner har skiftat riktning?\""
+    other = "Vad menas med att \"downstream-partnern har bytt inriktning?\""
```
Comment (SergeCroise) [inline i18n/sv.toml:73]: other = "Vad menas med att \"downstream-partnern har bytt inriktning\"?"

File: i18n/sv.toml [hunk 2]
```diff
@@ -38,47 +38,49 @@ other = "sv"
 
 [copyright]
   [copyright.line1]
-  other = "© 2020 Rocky Linux-projektet. Alla rättigheter förbehållna."
+  other = "© 2020 Rocky Linux projektet. Alla rättigheter förbehållna."
```
Comment (bittin) [inline i18n/sv.toml:41]: Looks good

File: i18n/sv.toml [hunk 3]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
 
   [copyright.line2]
-  other = "CentOS är ett registrerat varumärke som tillhör Red Hat, Inc. Rocky Linux-projektet är inte anslutet till eller godkänt av Red Hat, Inc."
+  other = "CentOS är ett registrerat varumärke som tillhör Red Hat, Inc. Rocky Linux projektet är inte anknutet till eller godkänt av Red Hat, Inc."
```
Comment (bittin) [inline i18n/sv.toml:44]: Looks good

File: i18n/sv.toml [hunk 4]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
 
 [errors]
   [errors.404]
     [errors.404.title]
     other = "Sidan hittades inte."
 
     [errors.404.explanation]
-    other = "Den sida du försökte nå kunde inte hittas. Var god försök igen."
+    other = "Sidan du försökte nå kunde inte hittas. Var god försök igen."
```
Comment (bittin) [inline i18n/sv.toml:52]: Looks good

File: i18n/sv.toml [hunk 5]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
 
 [landing]
   [landing.p1.heading]
   other = "Rocky Linux"
 
   [landing.p1.content]
-  other = "En gemenskapsdriven insats för att ge dig en produktionsklar Linux i företagsklass."
+  other = "En community-driven insats för att skapa en enterprise-klassad linuxdistribution redo att användas i skarp drift."
```
Comment (bittin) [inline i18n/sv.toml:59]: "En gemenskapsdriven insats för att skapa en enterprise-klassad linuxdistribution redo att användas i skarp drift."
Yeah enterprise-klassad och företagsklassad not sure whats best?

File: i18n/sv.toml [hunk 6]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
 
   [landing.p2.heading]
-  other = "Vad är Rocky Linux Projektet?"
+  other = "Vad är Rocky Linux projektet?"
```
Comment (bittin) [inline i18n/sv.toml:62]: Looks good

File: i18n/sv.toml [hunk 7]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
 
   [landing.p2.content]
-  other = "Rocky Linux är ett gemenskapsdrivet operativsystem utformat för att vara 100% bugg-för-bugg-kompatibelt med Amerikas främsta företagsklass Linux distribution nu när dess nedströms partner har skiftat inriktning. Det är under intensiv utveckling av gemenskapen. Rocky Linux leds av Gregory Kurtzer, grundare av CentOS-projektet. Det finns inget datum satt för första utgåvan. Bidragsgivare uppmanas att nå oss med hjälp av de kommunikationsalternativ som erbjuds på denna webbplats."
+  other = "Rocky Linux är ett community-drivet enterprise-operativsystem designat för att vara 100% bugg-för-bugg kompatibelt med Amerikas främsta enterprise-klassade linuxdistribution nu när downstream-partnern har bytt inriktning. Rocky Linux är under intensiv utveckling av communityn. Det leds av Gregory Kurtzer, grundare av CentOS-projektet. Det är ännu inte bestämt när första versionen ska släppas. Ni som vill bidra uppmanas att kontakta oss med hjälp av de kommunikationsalternativ som erbjuds på webbplatsen."
```
Comment (bittin) [inline i18n/sv.toml:65]: Gemenskapsdriven is more correct Swedish also not sure if Enterprise or Företag is best?

File: i18n/sv.toml [hunk 8]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
+ 
+[faq]
```
Comment (bittin) [inline i18n/sv.toml:67]: *Approve*

File: i18n/sv.toml (hunk 1 above)
Comment (bittin) [inline i18n/sv.toml:73]: Looks good

File: i18n/sv.toml [hunk 9]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
   other = "Vanliga frågor"
 
   [faq.q]
     [faq.q.downstream-partner-direction]
-    other = "Vad menar ni med att \"dess nedströmspartner har skiftat riktning?\""
+    other = "Vad menas med att \"downstream-partnern har bytt inriktning?\""
 
     [faq.q.come-in]
-    other = "Så var kommer Rocky Linux in?"
+    other = "Så var kommer Rocky Linux in i bilden?"
```
Comment (bittin) [inline i18n/sv.toml:76]: Looks good

File: i18n/sv.toml [hunk 10]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
 
   [faq.a]
     [faq.a.downstream-partner-direction]
-    other = "CentOS-projektet tillkännagav nyligen ett skifte i strategi för CentOS. Medan CentOS tidigare existerade som ett nedströmsbygge av dess uppströmsleverantör (det tar emot korrigeringar och uppdateringar efter att uppströmsleverantören gör det) kommer den att flyttas till att bara bli ett uppströmsbygge (testa korrigeringar och uppdateringar innan de inkluderas i uppströmsleverantören). Dessutom har stödet för CentOS Linux 8 förkortats, från 31 maj 2029 till 31 december 2021."
+    other = "CentOS-projektet tillkännagav nyligen att de har bytt strategi för CentOS. Tidigare existerade CentOS som en downstream build från upstream vendorn (den tar emot patcher och uppdateringar efter att upstream vendorn gör det) men i och med att de har bytt till att vara en upstream build (där patcher och uppdateringar testas innan de inkluderas i upstream vendorn). Dessutom har stödet för CentOS Linux 8 förkortats, från 31 maj 2029 till 31 december 2021."
```
Comment (bittin) [inline i18n/sv.toml:80]: Looks good

File: i18n/sv.toml [hunk 11]
```diff
@@ -38,47 +38,49 @@ other = "sv"
...
 
     [faq.a.come-in]
-    other = "Rocky Linux mål är att fungera som ett nedströmsbygge som CentOS har gjort tidigare, att bygga utgåvor efter de har lagts till av uppströmsleverantören, inte före."
+    other = "Målet med Rocky Linux är att fungera som en downstream build som CentOS tidigare gjorde, där releaser byggdes efter de har lagts till av upstream vendorn, inte före."
```
Comment (bittin) [inline i18n/sv.toml:83]: Looks good

**Review by bittin — COMMENTED** [review]
Did a review of it now

**Comment by bittin** [comment]
Did a review now

File: i18n/sv.toml (hunk 5 above)
Comment (cascading-jox) [reply i18n/sv.toml:59]: I have never heard the word företagsklass and I don't know what it means. Rewriting it like this "klass för företag" maybe makes my confusion more obvious. What do you think?

File: i18n/sv.toml (hunk 7 above)
Comment (cascading-jox) [reply i18n/sv.toml:65]: I guess it's a matter of taste. Maybe I'm turning into an anglo-saxon but I think community makes more sense than gemenskap, in software development at least. I believe gemenskap would be more equal to fellowship.

File: i18n/sv.toml (hunk 1 above)
Comment (SergeCroise) [inline i18n/sv.toml:73]: other = "Vad menas med att \"downstream-partnern har bytt inriktning\\"?"

Comment (SergeCroise) [inline i18n/sv.toml:73]: other = "Vad menas med att \\"downstream-partnern har bytt inriktning\\"?"

**Review by SergeCroise — CHANGES_REQUESTED** [review]

File: i18n/sv.toml (hunk 1 above)
Comment (bittin) [reply i18n/sv.toml:73]: > other = "Vad menas med att "downstream-partnern har bytt inriktning\"?"

even better

**Comment by hbjydev** [comment]
I'm now at a point where I don't want to accept any more i18n PRs, simply because it's going to become FAR too unmaintainable, so I'm going to prioritise #134 and close this PR.

**Pull request closed without merging**